src/Traits/DiffableEntityTrait.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Equip\Data\Traits;
+
+/**
+ * Intended to be used by classes implementing Equip\Data\EntityInterface that
+ * also need to implement Equip\Data\DiffableInterface.
+ */
+trait DiffableEntityTrait /* implements DiffableInterface */
```

Comment:
Yup. I can remove the comment, but I thought it might be useful for whoever reads the code, since if this is merged we will have both it and `DiffableTrait` at that point.